### ISSUE TITLE
"Connected Applications" list stucks at loading state

### ISSUE THREAD

**Issue by crazytonyli** [Automattic/wp-calypso#96037]
### Quick summary

[The Connected Applications page](https://wordpress.com/me/security/connected-applications) (and the "Connections" section at the bottom of [Security page](https://wordpress.com/me/security)) does not display the applications list.

### Steps to reproduce

Go to https://wordpress.com/me/security/connected-applications.

### What you expected to happen

After a while, the list will change from the loading state to a result state (empty or a list of applications).

### What actually happened

The web page stucks at the loading state forever.

### Impact

Some (< 50%)

### Available workarounds?

No and the platform is unusable

### Logs or notes

This issue may not be reproducible on all accounts. It works as expected on my A8C account. But it never works on my test account. I'm happy to provide my test account for debugging. Let me know on Slack if you need it.

**Comment by taipeicoder** [comment]
I'm not able to reproduce this on my end. Do you see any error in the browser devtool console or network request?

**Comment by crazytonyli** [comment]
@taipeicoder No. The `connected-applications` API call returns 200. And, I can see the applications in the API responses.

**Comment by lsl** [comment]
Couldn't repro, removing blocker label for now.

**Comment by lsl** [comment]
Credentials for user id 223970963 are in secret store under "Test account for wp-calypso #96037". Could repro using SU, can't see any errors. Taking a closer look.

**Comment by lsl** [comment]
There's a few warnings on this page in development but I suspect it's related to this validation error, ![Image](https://github.com/user-attachments/assets/5af5ac63-e9f5-4666-a51c-ebe65f600ce5)
